src/main/java/dk/kb/util/yaml/YAML.java

```diff
@@ -694,11 +704,23 @@ public static YAML resolveMultiConfig(String... configNames) throws IOException
             }
         }
     }
+    private static InputStream openStream(Path path) {
+        try {
+            System.out.println("Opening " + path);
```

Comment:
Replace with log.debug perhaps?

Reply:
Nah, it already log.debugs the list of resolved paths. It was just a leftover from locating a problem.